src/Microsoft.VisualStudio.Editors/OptionPages/GeneralOptionPageResources.resx

```diff
@@ -126,6 +126,9 @@
   <data name="General_FastUpToDateCheck_LogLevel_Info" xml:space="preserve">
     <value>Info</value>
   </data>
+  <data name="General_FastUpToDateCheck_LogLevel_Minimal" xml:space="preserve">
+    <value>Minimal</value>
+  </data>
```

Comment:
Are we still able to meet localization deadline? @tmeschter

Reply:
This is going into master and master is, to my knowledge, a 16.4 branch; the localization deadline doesn't apply.

Reply:
Yes, this is for 16.4.